astropy_healpix/conftest.py

```diff
@@ -1,61 +1,12 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-# this contains imports plugins that configure py.test for astropy tests.
-# by importing them here in conftest.py they are discoverable by py.test
-# no matter how it is invoked within the source tree.
-
-from astropy.tests.pytest_plugins import *
```

Comment:
only change this line

Reply:
(with the additions below, but also don't forget to import `enable_deprecations_as_exceptions`, the next line in the template after the try/except